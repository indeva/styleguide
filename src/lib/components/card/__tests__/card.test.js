import React from 'react';
import renderer from 'react-test-renderer';
import Card from '..';

describe('Card', () => {
  it('should render card properly', () => {
    const component = renderer.create(<Card>my card</Card>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  describe('with featured modifier', () => {
    it('should render card properly', () => {
      const component = renderer.create(<Card featured>small link</Card>);
      const tree = component.toJSON();
      expect(tree).toMatchSnapshot();
    });
  });

  describe('whith custom class', () => {
    it('should render card properly', () => {
      const component = renderer.create(<Card className="my classes">my card</Card>);
      const tree = component.toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
