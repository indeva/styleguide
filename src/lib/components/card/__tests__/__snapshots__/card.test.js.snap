// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should render card properly 1`] = `
<div
  className="card"
>
  my card
</div>
`;

exports[`Card whith custom class should render card properly 1`] = `
<div
  className="card my classes"
>
  my card
</div>
`;
